src/gene/database/database.py

```diff
@@ -296,7 +296,7 @@ def create_db(
     >>>
     >>> os.environ["GENE_NORM_ENV"] = "Prod"
     >>> prod_db = create_db()  # creates connection to AWS cloud DynamoDB instance,
-    >>>                        # overruling `GENE_NORM_DB_URL` variable setting
+    >>> # overruling `GENE_NORM_DB_URL` variable setting
```

Comment:
I think this actually ends up being kind of confusing to read. can we just move the comment up to the line above `prod_db = create_db()` so that it fits all on one line?